src/components/BudgetCmpt.vue

```diff
@@ -31,7 +31,35 @@
     </div>
     <div class="scrollable">
       <div id="budgetTables">
-        <div class="placeholderTop" ></div>
+        <div class="placeholderTop">
+          <div id="budgetHeader">
+            <div class="row date">
+              <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-left"/></div>
+              <div class="col-8 date-label" :class="this.toBeBudgetedClass">
+                <p class="month">{{ $d(this.getMonthAsDate(budgetMonth), 'monthString') }} <span v-if="!this.isThisYear"> {{ $d(this.getMonthAsDate(budgetMonth), 'year') }}</span></p>
+                <p class="toBeBudgeted" v-if="this.toBeBudgeted > 0"> {{ getEurosAmount(this.toBeBudgeted) }} € {{$t('TO_BE_BUDGETED')}}</p>
+                <p class="toBePulledOut" v-else-if="this.toBeBudgeted < 0"> {{ getEurosAmount(-1*this.toBeBudgeted) }} € {{$t('TO_BE_PULLED_OUT')}}</p>
+              </div>
+              <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-right" /></div>
+            </div>
+            <table id="totalTable" class="table">
+              <tr>
+                <th class="col-6 name"></th>
+                <th class="col-2 allocated"><div>{{ $t("ALLOCATED") }}</div></th>
+                <th class="col-2 spent"><div>{{ $t("SPENT") }}</div></th>
+                <th class="col-2 available"><div>{{ $t("AVAILABLE") }}</div></th>
+              </tr>
+              <tbody>
+                <tr>
+                  <td class="name"><div>{{ $t("TOTAL") }}</div></td>
+                  <td class="allocated">{{ getEurosAmount(this.totalBudgetData.allocated) }}</td>
+                  <td class="spent">{{ getEurosAmount(this.totalBudgetData.spent) }}</td>
+                  <td class="available">{{ getEurosAmount(this.totalBudgetData.available) }}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
```

Comment:
not sure what happened here. Where was the whole budgetHeader before you add it here ?

Reply:
placeholderTop must have a changing height (money to distribute or not), it the same part than div .fixed to be sure it's the same height and not having bugs.